src/ui/server/controllers/insurance/policy/broker-manual-address/save-and-back/index.ts

```diff
@@ -0,0 +1,49 @@
+import { Request, Response } from '../../../../../../types';
+import { ROUTES } from '../../../../../constants';
+import constructPayload from '../../../../../helpers/construct-payload';
+import generateValidationErrors from '../validation';
+import mapAndSave from '../../map-and-save/broker';
+import { FIELD_ID } from '..';
+
+const { INSURANCE_ROOT, ALL_SECTIONS, PROBLEM_WITH_SERVICE } = ROUTES.INSURANCE;
+
+/**
+ * saves and goes back to all sections from Broker manual address page unless there are API errors
```

Comment:
do we need capital s here? :P

Reply:
We have 19 instances of this, can align in another PR/ticket. Have created EMS-4018.